tests/cache/models/samples/get.js

```diff
@@ -76,6 +80,80 @@ describe('tests/cache/models/samples/get.js, ' +
     });
   });
 
+  it('get with wildcard and with cacheGetSamplesWildcard flag on ' +
+    'should cache response', (done) => {
+    tu.toggleOverride('cacheGetSamplesByNameWildcard', true);
```

Comment:
toggle override here without storing original value and resetting it back to that value